client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -128,14 +128,6 @@ body {
   }
 }
 
-.add-widget-container {
-  background: fade(@redash-gray, 10%);
-  border-radius: @redash-radius;
-  margin-top: 15px;
-  //box-shadow: fade(@redash-gray, 15%) 0px 4px 9px -3px;
-  border: 2px dashed fade(@redash-gray, 25%);
-}
```

Comment:
@kocsmy I'm not sure if this is really not needed, but after rebasing with latest master this part was making the background grey & transparent (you could see the footer behind the add-widget section). I merged the edit flow PR, but please review the CSS in the final state to make sure the rebase didn't mess up something...

Reply:
Thanks for removing that part, it was indeed just a leftover from previous iterations. I'll review the CSS to make sure everything is in place.

Reply:
Ok, I reviewed it and all seems good now. 👌